src/Simplex/Chat.hs

```diff
@@ -5917,6 +5942,11 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
             XGrpInfo p' -> xGrpInfo gInfo author p' rcvMsg msgTs
             _ -> messageError $ "x.grp.msg.forward: unsupported forwarded event " <> T.pack (show $ toCMEventTag event)
 
+    xGrpMsgSkipped :: GroupInfo -> GroupMember -> SharedMsgId -> RcvMessage -> UTCTime -> CM ()
+    xGrpMsgSkipped _gInfo _m _sharedMsgId _msg _brokerTs = do
+      -- TODO can request delivery starting with _sharedMsgId here
+      pure ()
```

Comment:
originally had a new type of group event chat item created here, but removed it as there's already integrity error item created so it seems unnecessary. Maybe should add it back, with it saying "member continued delivery for you"